src/atomicdex/app.cpp

```diff
@@ -947,8 +953,8 @@ namespace atomic_dex
         });
         if (res)
         {
-            addressbook_model* addressbook = qobject_cast<addressbook_model*>(m_manager_models.at("addressbook"));
-            addressbook->initializeFromCfg();
+            system_manager_.get_system<addressbook_manager>().load_configuration();
```

Comment:
same here, i think `addressbook_page` will subscribe to `mm2_initialized` and will load configuration itself, this will clear app.cpp a bit, our goal is to have less code possible into this file.

This means that `addressbook_page` will have a `connect_signals` and `disconnect_signals` function same as `trading_page` please take a look.